src/perfEvents_linux.cpp

```diff
@@ -57,6 +57,7 @@ enum {
     HW_BREAKPOINT_X  = 4
 };
 
+int PerfEvents::_target_cpu = 0;
```

Comment:
Place it next to other PerfEvents members in this file.

Reply:
52bccf8791352d47a0bd3e01336f412e5aba43c2